dist/fonts/pbds-sports.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
this font is for the docs only, (maturity matrix), should be removed (all formats)